### PR TITLE
EUI-7733 Reserved to judge component not being prepopulated

### PR DESCRIPTION
**Pull request by johnbenjamin-hmcts**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-7733

### Change description ###
Reserved to judge component not being prepopulated

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: projects/ccd-case-ui-toolkit/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@hmcts/ccd-case-ui-toolkit",
-  "version": "6.13.10-rc1",
+  "version": "6.10.12-judicial-user-typeahead-fix",
```
Comment (GaryScottHMCTS) [inline projects/ccd-case-ui-toolkit/package.json:3]: why has the version been reduced

File: projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/judicial-user/write-judicial-user-field.component.spec.ts [hunk 2]
```diff
@@ -104,7 +124,9 @@ describe('WriteJudicialUserFieldComponent', () => {
       schemas: [CUSTOM_ELEMENTS_SCHEMA],
       providers: [
         { provide: ActivatedRoute, useValue: activatedRoute },
```
Comment (GaryScottHMCTS) [inline projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/judicial-user/write-judicial-user-field.component.spec.ts:126]: is ActivatedRoute still needed?

File: projects/ccd-case-ui-toolkit/package.json (hunk 1 above)
Comment (johnbenjamin-hmcts) [reply projects/ccd-case-ui-toolkit/package.json]: @GaryScottHMCTS - thank you for pointing this out, I have now updated the version.

File: projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/judicial-user/write-judicial-user-field.component.spec.ts (hunk 2 above)
Comment (johnbenjamin-hmcts) [reply projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/judicial-user/write-judicial-user-field.component.spec.ts]: I have removed the ActivatedRoute as it is no longer required. Thank you.

**Review by GaryScottHMCTS — APPROVED** [review]

**Review by Kartheek-S — APPROVED** [review]

**Pull request merged**